config/sets/doctrine-orm-28.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Rector\Doctrine\Orm28\Rector\MethodCall\IterateToToIterableRector;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->rules([
+        IterateToToIterableRector::class,
```

Comment:
Could you add comment to link Doctrine upgrade.md/changelog/PR on Github? Just so we have a note about the change right in the set

Reply:
Good point added the comment and made it a one line `rule()`